src/containers/PatientProfilePage.tsx

```diff
@@ -0,0 +1,243 @@
+import React, { useState, useEffect } from "react";
+import "../styles/EventCreationPage.css";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import FormField from "../components/common/FormField";
+import CompletePatientButton from "../components/PatientCreationPage/CompletePatientButton";
+import RadioSelector from "../components/common/RadioSelector";
+import TriagePills from "../components/PatientCreationPage/TriagePills";
+import StatusPills from "../components/PatientCreationPage/StatusPills";
+import { useMutation } from "@apollo/react-hooks";
+import { useQuery } from "react-apollo";
+import {
+  triageLevel,
+  status,
+  PatientType,
+  GET_PATIENT_BY_ID,
+  GET_ALL_PATIENTS,
+} from "../graphql/queries/templates/patients";
+import {
+  ADD_PATIENT,
+  EDIT_PATIENT,
+} from "../graphql/mutations/templates/patients";
+
+interface FormFields {
+  barcodeValue: string;
+  triage: triageLevel | null;
+  gender: string;
+  age: number | null;
+  notes: string;
+  runNumber?: number | null;
+  collectionPointId?: number;
+  status: status | null;
+  triageCategory?: number | null;
+  triageLevel?: number | null;
+  transportTime?: number | null;
+}
+
+const PatientProfilePage = ({
+  match: {
+    params: { mode, ccpId, patientId },
+  },
+}: {
+  match: { params: { mode: string; patientId?: string; ccpId: string } };
+}) => {
+  const { data, loading, error } = useQuery(
+    mode === "edit" && patientId
+      ? GET_PATIENT_BY_ID(patientId)
+      : GET_ALL_PATIENTS
+  );
```

Comment:
This is causing a network request error with message "Unknown directive "client"."
I think it's because you haven't called `useAllPatients()` anywhere.

Reply:
This is actually related to the cache. A similar problem can be found in `/events`. This should be added into the offline ticket.

Reply:
yep, this can be left in for now